sys/kern/src/arch/riscv32.rs

```diff
@@ -621,45 +621,24 @@ unsafe fn handle_fault(task: *mut task::Task, fault: FaultInfo) {
 // We currently only support single HART systems.  From reading elsewhere,
 // additional harts have their own mtimecmp offset at 0x8 intervals from hart0.
 //
-// As per FE310-G002 Manual, section 9.1, the address of mtimecmp on
-// our supported board is 0x0200_4000, which also matches qemu.
-//
-// On both RV32 and RV64 systems the mtime and mtimecmp memory-mapped registers
-// are 64-bits wide.
-//
-
 // Configure the timer.
 //
 // RISC-V Privileged Architecture Manual
 // 3.2.1 Machine Timer Registers (mtime and mtimecmp)
 //
-// To keep things simple, especially on RV32 systems where we cannot atomically
-// write to the mtime/mtimecmp memory-mapped registers as they are 64 bits
-// wide, we only utilise the first 32-bits of each register, setting the
-// high-order bits to 0 on startup, and restarting the low-order bits of mtime
-// back to 0 on each interrupt.
-//
 #[no_mangle]
```

Comment:
Does this function need to be marked `no_mangle`?

Reply:
It's not exported so probably not.

Reply:
Nope, removed.